docs/architecture/adr-047-extend-upgrade-plan.md

```diff
@@ -42,63 +42,30 @@ Currently, there is no mechanism that makes Cosmovisor run a command after the u
 The current upgrade process has this timeline:
 
 1. An upgrade governance proposal is submitted and approved.
-1. The upgrade height is reached.
-1. The `x/upgrade` module writes the `upgrade_info.json` file.
-1. The chain halts.
-1. Cosmovisor backs up the data directory (if set up to do so).
-1. Cosmovisor downloads the new executable (if not already in place).
-1. Cosmovisor executes the `${DAEMON_NAME} pre-upgrade`.
-1. Cosmovisor restarts the app using the new version and same args originally provided.
+2. The upgrade height is reached.
+3. The `x/upgrade` module writes the `upgrade_info.json` file.
+4. The chain halts.
+5. Cosmovisor backs up the data directory (if set up to do so).
+6. Cosmovisor downloads the new executable (if not already in place).
+7. Cosmovisor executes the `${DAEMON_NAME} pre-upgrade`.
+8. Cosmovisor restarts the app using the new version and same args originally provided.
 
 ## Decision
 
 ### Protobuf Updates
 
-We will update the `x/upgrade.Plan` message for providing upgrade instructions.
-The upgrade instructions will contain a list of artifacts available for each platform.
-It allows for the definition of a pre-run and post-run commands.
-These commands are not consensus guaranteed; they will be executed by Cosmosvisor (or other) during its upgrade handling.
+The `x/upgrade.Plan` message is updated for providing upgrade artifacts. The artifacts available define each platform.
```

Comment:
Since this is discussing things not yet done, it should be future tense. Also, the second sentence is weird to me. I'm not quite sure what it's trying to say.

```suggestion
The `x/upgrade.Plan` message will be updated to provide a list of artifacts, one for each available platform.
```